ts/image-occlusion/template.ts

```diff
@@ -0,0 +1,202 @@
+export const IOQFMT = `{{#Image}}
```

Comment:
This card templates `qfmt`, `afmt` and `css`. How can css be added to card templates?

Reply:
when creating notetype:

```
        NotetypeConfig {
            css: "foo".to_string(),
            ..Default::default()
        }
```

Reply:
Thanks